symengine/cwrapper.cpp

```diff
@@ -58,6 +58,25 @@ inline bool is_aligned(T *p, size_t n = alignof(T))
 
 extern "C" {
 
+
+#define CWRAPPER_BEGIN()                                                       \
+    try                                                                        \
+    {
+
+#define CWRAPPER_END(error, code)                                              \
+        return 0;                                                              \
+    }                                                                          \
+    catch ( error )                                                            \
+    {                                                                          \
+        return code;                                                           \
+    }
+
+#define CWRAPPER_END_EXTRA(error, code)                                        \
```

Comment:
`DivisionByZero` is not found in only `div`, it can be thrown in any of the other functions that call `div`